### PR TITLE
Add command to make deposit

### PR DESCRIPTION
**Pull request by im-kulikov**
Add command that makes a deposit for the default wallet

### CODE REVIEW

**Comment by realloc** [comment]
I propose to convert it to a script without hardcoded addresses. It has access to chain, contracts and wallets it can get all required information by itself.

File: services/chain/artifacts.mk [hunk 1]
```diff
@@ -13,4 +13,15 @@ else
...
 endif
 
-
+# Makes deposit for default wallet
+deposit:
+	@docker exec -it main_chain \
+		neo-go contract invokefunction \
+			-w wallets/wallet.json \
+			-a NTrezR3C4X8aMLVg7vozt5wguyNfFhwuFx \
+			-r http://localhost:30333 \
```
Comment (realloc) [inline services/chain/artifacts.mk:22]: -r http://main_chain.neofs.devenv:30333 \

Comment (im-kulikov) [reply services/chain/artifacts.mk]: fixed

**Comment by im-kulikov** [comment]
> I propose to convert it to a script without hardcoded addresses. It has access to chain, contracts and wallets it can get all required information by itself.

1. As for the wallet, I disagree, because the wallet must be forwarded and have a hard-coded path
2. Node address, we have one, and therefore there is no point in configuring it
3. This method is supposed to be used for testing and development of applications and/or services, and therefore the recipient of the payment - ʻuser.key`, we know in advance

In the form of a script, it will look quite cumbersome, and perhaps it will make sense in the future, but for now, I would stop at a small make command

сс @alexvanin @pkorotkov

**Review by pkorotkov — APPROVED** [review]

**Review by realloc — CHANGES_REQUESTED** [review]
- Hardcoding neo node url breaks domain settings
- Putting this action to `artifacts.mk` breaks `Makefile` separation logic

Let me show what I mean in next commit

**Comment by im-kulikov** [comment]
> * Hardcoding neo node url breaks domain settings
> * Putting this action to `artifacts.mk` breaks `Makefile` separation logic
> 
> Let me show what I mean in next commit

pls wait until I'll upload the latest version agreed with @alexvanin

File: .env [hunk 2]
```diff
@@ -16,3 +16,5 @@ IR_IMAGE=nspccdev/neofs-ir
 # NeoFS Storage nodes
 NODE_VERSION=0.12.0-rc1
 NODE_IMAGE=nspccdev/neofs-storage
+
+NEOFS_CONTRACT_MAINCHAIN=af5dc5f7e6a6efc64d679098f328027591a2e518
```
Comment (realloc) [inline .env:20]: It's already in `services/ir/.ir.env`. Common .env should not have volatile serivice-specific settings.

Comment (im-kulikov) [reply .env:20]: yeah, but it contains IR specific environments, that should not be accessible by a another services, isn't it?

**Comment by realloc** [comment]
Closed by #9

**Pull request closed without merging**